content/en/service_management/on-call/guides/migrating-from-your-current-providers.md

```diff
@@ -0,0 +1,155 @@
+
+### Monitor the migration
+
+Use Datadog dashboards to observe migration performance in real time. Watch for:
+
+- Alert volume by provider
+- Acknowledgment and escalation latency
+- Incidents lacking team ownership
+
+These signals help validate readiness, detect misconfigurations, and surface issues early before the full cutover.
+
+## Cut over and retire legacy systems
+
+After validation is complete and all teams are actively using Datadog On-Call, begin phasing out your legacy provider. To minimize disruption, most teams approach this incrementally:
+
+- Retire low-severity or infrequent alerting paths first
+- Remove deprecated schedules, escalation policies, and routing keys
+- Archive legacy configurations or export them as documentation for reference
+
+Double-check that all monitors point exclusively to Datadog On-Call and that legacy integrations are no longer in use. If your dual-routing period revealed any inconsistencies or gaps, address those before finalizing the cutover.
+
+Completing this step ensures a clean transition and eliminates the risk of confusion or missed alerts during incident response.
+
+## Sustain and scale your on-call practice
+
+With your core migration to Datadog On-Call complete, shift focus to long-term operations and continuous improvement. Use the following practices to keep your on-call processes healthy, maintain team readiness, and evolve your setup as your needs grow.
+
+- **Establish ongoing ownership**: Assign clear ownership for Datadog On-Call within your team. This includes maintaining schedules, onboarding new responders, and adapting to feature changes over time.
+- **Incorporate post-mortems**: Review incidents that occurred during or after the migration to identify any missed escalation or alerting issues. Feed those lessons back into your testing and runbook documentation.
+- **Track on-call health**: Use [On-Call Analytics][8] to monitor alert volume per responder, MTTA/MTTR trends, notification fatigue, and recurring escalations.
+- **Stay current**: Subscribe to [Incident Response product updates][9] to keep up with new features, improvements, and deprecations.
+- **Deepen your product knowledge**: Explore Datadog's documentation on [Incident Management][6], [Schedules][7], and [Integrations][2] to expand your use of the platform.
+- **Join the community**: Connect with peers and Datadog engineers in the [Datadog Slack Community][10] to share best practices, get advice, and give feedback.
+- **Schedule a retrospective**: Within 30-60 days post-migration, hold a retrospective to capture lessons learned and update your documentation, internal guides, and test plans.
+
+## Further reading
+
+{{< partial name="whats-next/whats-next.html" >}}
+
+[1]: https://app.datadoghq.com/on-call/
+[2]: /integrations/
+[3]: /monitors/notify/#bulk-editing-monitor--handles
+[4]: /on_call/teams/
+[5]: https://www.google.com
```

Comment:
The text for this link mentions a migration tool, but that actual link takes you to the main Google page. Do you have a link to the migration tool that I can add?

Reply:
@Dalje-et Thanks for clarifying. The other guide will also need to be reviewed. Are you okay with waiting to publish this guide until after the migration tool guide has been reviewed and published?

Reply:
I took a closer look at the other PR. In this guide, we refer to a "dedicated migration tool", but the tool is really more of a step-by-step guide. I think we should update the paragraph where we mention the tool and instead refer to it as a guide. I'd also advocate for converting it into a note, as it's more of a callout vs. a required step.